Kubernetes/deploy/terraform/modules/istio-inject-headers/istio.tf

```diff
@@ -86,7 +68,9 @@ resource "kubernetes_manifest" "authorization_policy_backend" {
             {
               source = {
                 principals = [
-                  "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"
+                  "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account",
```

Comment:
I would check the service account names! I don't think they are the right ones. You can check [here](https://github.com/enthought/terraform-kubernetes-local-bootstrap/blob/main/variables.tf#L33) or easier directly with k9s